packages/trackerless-network/src/logic/StreamEntryPointDiscovery.ts

```diff
@@ -192,13 +203,23 @@ export class StreamEntryPointDiscovery {
                 const rediscoveredEntrypoints = await this.discoverEntryPoints(streamPartId)
                 await stream.layer1!.joinDht(rediscoveredEntrypoints, false, false)
                 if (stream.layer1!.getBucketSize() < NETWORK_SPLIT_AVOIDANCE_LIMIT) {
+                    const nodesToAvoid = rediscoveredEntrypoints.filter((peer) => !stream.layer1!.getKBucketPeers().includes(peer))
+                    this.setAvoidedNodes(streamPartId, nodesToAvoid)
```

Comment:
Does this overwrite as the `set` prefix tells? Or could rename to `addAvoidedNodes`?